modules/cluster-and-ha-management/pages/set-up-crr.adoc

```diff
@@ -8,12 +8,6 @@ Changes on the primary cluster are copied over to the DR cluster.
 
 When necessary, you can fail over to a DR cluster, making it the new primary cluster.
 
-[NOTE]
-====
-This setup assumes that you are setting up a DR cluster for an existing primary cluster.
-If you are setting up both the primary cluster and DR cluster from scratch, you only need perform Step 3 after TigerGraph is installed on both clusters.
-====
-
```

Comment:
We recommend the same process for enabling PR and DR clusters, whether they are built from scratch or not.